### DISCUSSION TITLE
TFT35 V2.0 dead after bootloader

### DISCUSSION THREAD

**Discussion by jtrucker4** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2119] [Q&A]
Bought a BTT 1.4 Turbo with TFT35 V2.0 bundle last summer and finally getting around to building printer.

Went to flash most recent firmware into TFT35 and it wouldn't load.  So bought a CP2102 module, used FlyMcu, and flashed the TFT35_V2.0_bootloader posted here: https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware.  Text log from FlyMcu below.

Now display shows "illegal user App1!" when powered up, and no change if SD card with new firmware is inserted.  Is it dead, or was it the wrong bootloader for this board?  How to get it to load firmware?

Thanks!

...Delay 100ms
DTR be Low(-3--12V),Reset Released
RTS keep High
Connectting...5, Received:1F 1F 
Connect Ok @COM4@115200bps,@1297ms
BootLoader Version:2.2
PID:00000414  STM32F10xxx_High-density
OptionBytes readout:
A55AFF00FF00FF00FF00FF00FF00FF00
96bit unique device ID:
[36FFD8054157333144741357]
[05D8FF36 31335741 57137444]
FLASH ROM size:256KB
SRAM size:65535KB(For reference only!!!)
@1672ms,Ready for Program
Write 10KB Ok,100%,@23000ms
Write Option Bytes: A5 5A FF 00 FF 00 FF 00 FF 00 FF 00 FF 00 FF 00
Write Option Bytes Ok
OptionBytes JustWrited:
A55AFF00FF00FF00FF00FF00FF00FF00
Go from 08000000 Failed...Maybe Writed OptionBytes just before!!!
www.mcuisp.com(Versatile Handy Programmer EP968,World's first):Mission Complete,Anything Ok!!!

**Comment by vicentegiuliani** [comment]
I have the same problem on the TFT35-V2.0, I bought it in July 2021 and the screen with GD32 chip was sent to me.

**Reply by jtrucker4** [reply]
Got a response to a msg I sent to BTT via Amazon that sorted it out.

From https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/tree/master/Copy%20to%20SD%20Card%20root%20directory%20to%20update  you'll want the following two files:
   BIQU_TFT35_APP1_V2.0.27.x.bin
   config.ini
   
And from the Language Packs folder there the relevant ini file (I used english).  Put all three files into the root directory of a FAT32 SD card and it should flash the firmware on the TFT35 V2.0.

I also used the TFT35 folder your favorite theme in the same repository, with folders and files inside, to add fonts and bmp icons as the process had wiped them.  My touchscreen also didn't work afterward, so made an empty reset.txt file in the root folder of the SD card to force a screen calibration and reenable touch.

All working now!
